miri-script/src/commands.rs

```diff
@@ -155,13 +155,15 @@ impl<'a> MiriRunner<'a> {
             Subcommands::Fmt { flags } => self.fmt(&flags),
             Subcommands::Clippy { flags } => self.clippy(&flags),
             Subcommands::Cargo { flags } => self.cargo(&flags),
-            Subcommands::ManySeeds { command, seed_start, seeds } =>
-                self.many_seeds(&command, *seed_start, *seeds),
+            Subcommands::ManySeeds { command, seed_start, seeds } => {
+                self.many_seeds(&command, *seed_start, *seeds)
+            }
```

Comment:
Yup, I use multiple setups so it might be the case. I am sorry, I'll try to keep it consistent.

Reply:
Whatever you are using to format your checkout should be able to pull in Miri's rustfmt.toml. Sometimes editors have difficulty with Miri or rustc because they aren't just standard Cargo projects. Can you take a skim over the CONTRIBUTING.md and see if anything there looks helpful? If there isn't, can you maybe open an issue or start a topic on the Zulip so we can figure out what Miri can do to improve this?